src/mongodb/scores.controller.ts

```diff
@@ -4552,13 +4552,12 @@ export class ScoresController {
 
       const url = process.env.ALL_CONTENT_SERVICE_API;
 
-      // Add the check for the limit
-      // Add the check for the limit
-      if (contentlimit < 5) {
-        contentlimit = 5;
-      } else if (contentlimit > 20) {
-        contentlimit = 20;
-      }
+      // Add the check for the limit for VAPT only.
+      // if (contentlimit < 5) {
+      //   contentlimit = 5;
+      // } else if (contentlimit > 20) {
```

Comment:
what if user send content limit as 100? will it send 100 content or 20 contents?